<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharGenSkills</class>
 <widget class="QWidget" name="CharGenSkills">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CharGenSkills</string>
  </property>
  <widget class="QLabel" name="bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>560</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>char_bg.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="bg_text11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>240</width>
     <height>252</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>text.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="bg_text2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>210</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>text.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="bg_text1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Waree</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>text.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="img1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>120</width>
     <height>180</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>character.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="img2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>120</width>
     <height>180</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>dragonborn.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="bg_text21">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>240</y>
     <width>240</width>
     <height>252</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>text.png</pixmap>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="nextStep">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>510</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Schoolbook L</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(136, 138, 133);
color: white
</string>
   </property>
   <property name="text">
    <string>  Старт</string>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="text1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Waree</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>               Игрок 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="text2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>210</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Waree</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>                 Игрок 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="skills11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>240</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Likhan&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="skills21">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>240</y>
     <width>240</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Likhan&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="skills12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>268</y>
     <width>240</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Likhan&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="skills13">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>296</y>
     <width>240</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Likhan&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="skills14">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>324</y>
     <width>240</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Likhan&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="skills15">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>352</y>
     <width>240</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Likhan&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="skills16">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>240</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Likhan&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="skills22">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>268</y>
     <width>240</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Likhan&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="skills23">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>296</y>
     <width>240</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Likhan&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="skills24">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>324</y>
     <width>240</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Likhan&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="skills25">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>352</y>
     <width>240</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Likhan&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="skills26">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>380</y>
     <width>240</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Likhan&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="skills17">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>408</y>
     <width>240</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="skills18">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>436</y>
     <width>240</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="skills19">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>464</y>
     <width>240</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="skills27">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>408</y>
     <width>240</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="skills28">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>436</y>
     <width>240</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="skills29">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>464</y>
     <width>240</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
